mava/components/jax/building/distributor.py

```diff
@@ -144,7 +144,10 @@ def on_building_launch(self, builder: SystemBuilder) -> None:
         Returns:
             None.
         """
-        builder.store.program.launch()
+        worker_manager = builder.store.program.launch()
+
+        if worker_manager:
+            builder.store.worker_manager = worker_manager
```

Comment:
Slight suggested tweak: have `builder.store.worker_manager = builder.store.program.launch()` and then inside system.py's `launch` just return `self._builder.store.worker_manager` instead of having the if statement for the returns.

Reply:
I think it will be a bit tough to do that since we can't easily access the builder from the launch function since we use the function this way `builder.store.program.launch() ` I tried to find a solution and I think it is possible but it will make the code more complex.
If you think this is necessary then I will be happy to change it!